docs/observability/aws/integrations/aws-ec2-metrics.md

```diff
@@ -4,159 +4,191 @@ title: AWS EC2 Metrics
+| json field=req_instancesSet "item", "items" as req_instancesSet_item, req_instancesSet_items nodrop
+| parse regex field=req_instancesSet_item "\"instanceId\":\s*\"(?<req_instanceid_3>.*?)\"" nodrop
+| parse regex field=req_instancesSet_items "\"instanceId\":\s*\"(?<req_instanceid_4>.*?)\"" nodrop
+| json field=responseElements "instancesSet.items" as res_responseElements_items nodrop
+| parse regex field=res_responseElements_items "\"instanceType\":\s*\"(?<res_instanceType>.*?)\"" nodrop
+| parse regex field=res_responseElements_items "\"instanceId\":\s*\"(?<res_instanceid>.*?)\"" nodrop
+| if (!isBlank(req_instanceid_1), req_instanceid_1,  if (!isBlank(req_instanceid_2), req_instanceid_2, if (!isBlank(req_instanceid_3), req_instanceid_3, if (!isBlank(req_instanceid_4), req_instanceid_4, "")))) as req_instanceid
+| if (!isBlank(req_instanceid), req_instanceid, res_instanceid) as instanceid
+| if (!isBlank(req_instancetype), req_instancetype, res_instancetype) as instanceType 
+| if (isEmpty(error_code), "Success", "Failure") as event_status
+| if (isEmpty(userName), user, userName) as user
+| tolowercase(instanceid) as instanceid
+| count as count by error_code | sort by count, error_code asc | limit 10
+```
 
-[Amazon Elastic Compute Cloud (Amazon EC2)](https://aws.amazon.com/ec2/) is a web service that provides secure, resizable compute capacity in the cloud, enabling you with complete control of your computing resources.
+## Install the AWS EC2 App and view the Dashboards
 
-The Sumo Logic AWS Observability EC2 Metrics Dashboards collects local host metrics and displays them using dashboards to aid with troubleshooting issues and to optimize resource allocation and costs. App dashboards provide a visual analysis of local host metrics for CPU, disk, memory, network, and TCP traffic.
+This page provides examples and descriptions for each of the AWS Observability AWS EC2 pre-configured dashboards.
 
-:::note
-We highly recommend you view these dashboards in the [Explore View](../deploy-use-aws-observability/view-dashboards.md) of the AWS Observability solution.
-:::
+The Sumo Logic App for AWS EC2 allows you to display your EC2 instance metrics (CloudWatch) using predefined dashboards. The App provides dashboards to display analysis of EC2 instance metrics for CPU, Disk, Network, EBS and Health Status Check. Also it provides detailed insights into all CloudTrail audit events associated with EC2 instances and specifically helps identify changes, errors, and user activities.
 
-### AWS EC2 Metrics - Overview
+### 1.1. AWS EC2 Overview (CloudWatch Metrics)
 
-The **AWS EC2 Metrics - Overview** dashboard provides insights into EC2 performance throughout your infrastructure, including CPU utilization, network rates, disk, and memory usage.
+The **AWS EC2 Overview (CloudWatch Metrics)** dashboard provides at-a-glance information about a EC2 CPU, instance disk store, network and EBS volume usage along with EC2 instance health status.
```

Comment:
Should this be "EC2 CPU instance disk storage"?

Reply:
CPU and Instance disk storage are different widgets. 
Updated this sentence for better clarity.

Reply:
I think this can be reworded as:
The AWS EC2 Overview (CloudWatch Metrics) dashboard provides at-a-glance information about EC2 CPU usage, instance disk storage, network usage, and EBS volume usage, as well as the health status of EC2 instances.